src/Feature/Editor/EditorSurface.re

```diff
@@ -302,9 +306,24 @@ let%component make =
       (0, 1);
     };
 
+  let%hook (scrollY, _setScrollYImmediately) =
+    Hooks.spring(
+      ~target=editor.scrollY,
+      ~restThreshold=100.,
+      ~enabled=smoothScroll,
+      scrollSpringOptions,
+    );
+  let%hook (scrollX, _setScrollXImmediately) =
+    Hooks.spring(
+      ~target=editor.scrollX,
+      ~restThreshold=100.,
```

Comment:
I used a high `restThreshold` to prevent a disconcerting 'jiggling' effect when the spring would be settling at the end of the animation. The font rendering would clamp to the nearest vertical pixel, with some lines being offset and some not transiently.